simple-examples/FileIO.hs

```diff
@@ -0,0 +1,174 @@
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
```

Comment:
Run this file through Ormolu to make the formatting consistent, and remove the redundant whitespace. Ormolu will choke on the linear arrows, because it doesn't support them, but you can easily enough preprocess/postprocess the source files and output as a workaround. cc @mrkkrp

Reply:
Done.